app/src/main/java/org/imozerov/streetartview/ui/detail/DetailArtObjectActivity.kt

```diff
@@ -126,6 +133,28 @@ class DetailArtObjectActivity : AppCompatActivity() {
         }
     }
 
+    private fun shareArtObjectInfo() {
+
+        val curImage: ImageView = art_object_detail_image
+                .findViewWithTag(GalleryPagerAdapter.TAG_CURRENT_DETAIL_IMAGE) as ImageView
+        val drawable = curImage.drawable
```

Comment:
We should encapsulate logic to get drawable from imageview. Maybe we can use extension function for it.